biz.aQute.bndlib/src/aQute/bnd/osgi/Processor.java

```diff
@@ -2633,4 +2633,18 @@ private List<File> getSelfAndAncestors(List<File> l) {
 	public void setPropertiesFile(File source) {
 		this.propertiesFile = source;
 	}
+
+	/**
+	 * Answer true if any of the headers is set as a property
+	 *
+	 * @param headers list of headers
+	 * @return true if any of the headers is set
+	 */
+	public boolean isHeaderSet(String... headers) {
```

Comment:
These are not all Headers. Some are instructions? Perhaps `isPropertySet` a better name for Processor?